Wrappers/Python/cil/optimisation/operators/FiniteDifferenceOperator.py

```diff
@@ -60,8 +60,12 @@ def __init__(self, domain_geometry,
            else:
                raise ValueError('Requested direction is not possible. Accepted direction is {} or {}, \ngot {}'.format(domain_geometry.dimension_labels, range(len(domain_geometry.shape)),  direction))
                          
-                                                                                                           
-        self.voxel_size = domain_geometry.spacing[self.direction]
+        #get voxel spacing, if not use 1s
```

Comment:
Add the same in the GradientOperator with c implementation, [here](https://github.com/TomographicImaging/CIL/blob/master/Wrappers/Python/cil/optimisation/operators/GradientOperator.py#L296)